e2e/app/aou-elements/xpath-defaults.ts

```diff
@@ -59,10 +59,11 @@ export function clickableXpath(label: string) {
  * @param label:
  * @param shapeValue:
  */
-export function clrIconXpath(label: string, shapeValue: string) {
-  if (label === '') {
+export function clrIconXpath(opts: TextOptions, shapeValue: string) {
```

Comment:
Is xpath the only way to reach this icon? I feel like it may be brittle in the face of future page reorganization and isn't the most readable representation for folks trying to investigate failed tests.

Also, I don't know what clr means.